_posts/2016/2016-11-28-xp-days-germany-2016.md

```diff
@@ -0,0 +1,120 @@
+This means that you are not able to separate the logic from the integration parts.
+Here the developer has to balance pros and cons whether mocking is not taking away any important implementation that should be tested.
+At the borders of your system mocking is indispensable, which means external services that are used have to be mocked.
+
+# Other workshops
+
+
+**[Impact Mapping](http://www.xpdays.de/2016/sessions/061-impact-mapping-workshop.html)**
+
+[Patrick Baumgartner](https://twitter.com/patbaumgartner) and [Peter Gfader](https://twitter.com/peitor) showed how important it is to take a step backward from your project to gain a deeper view.
+In general, an impact mapping workshop is done with all project stakeholders.
+It should deliver a common understanding of the problem that needs to be solved as well as clarify the impact that one (team/company) tries to achieve.
+By iterative answering of the most crucial business questions (why? who? how? what?) a non-technical perspective with a wide-spread solution horizon can be established.
+In this workshop small groups tried to answer these questions for an artificial business scenario, e.g. an advertisement company for sports events.
+With the given method and helpful hints by the instructors our team could figure out a vast palette of solutions to our problem space of which some also could avoid developing expensive software at all.
+In the end every participant was amazed about how many solution facets could be illuminated by this flexible and effective technique.
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="de" dir="ltr">Danke für eure Teilnahme an unserem Impact Mapping Workshop <a href="https://twitter.com/hashtag/xdde16?src=hash">#xdde16</a> <a href="https://twitter.com/hashtag/onstage?src=hash">#onstage</a> <a href="https://twitter.com/hashtag/fullhouse?src=hash">#fullhouse</a> <a href="https://twitter.com/hashtag/fun?src=hash">#fun</a> <a href="https://t.co/m5iXuzbKYe">pic.twitter.com/m5iXuzbKYe</a></p>&mdash; Patrick Baumgartner (@patbaumgartner) <a href="https://twitter.com/patbaumgartner/status/801044883855339520">November 22, 2016</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+**[Hacking Night](https://github.com/jlink/maexchen)**
+
+On monday night a two-hour coding contest took place.
+Small teams needed to write a [Mia game bot](https://github.com/jlink/maexchen), which interacted with the table server.
+Within four rounds the bots needed to compete against each other. It was a lot of fun and in the end our team bot could achieve the highest overall performance.
+
+# One more thing
+
+These days graphic recording of sessions is a major trend.
+Hence, there were not only 2 sessions for learning this skill yourself but also live sketching was done for of all sessions within the big hall by a professional artist – [Benjamin Felis](https://twitter.com/benjaminfelis).
```

Comment:
write out numbers and missing comma:
"Hence, there were not only two sessions for learning this skill yourself, but also..."